static/js/sensor_connection.js

```diff
@@ -1,10 +1,15 @@
+$("#sensorConnect").bootstrapSwitch({
+  onSwitchChange: function (e, state) {
+    alert(state);
+  },
+});
```

Comment:
add empty line after function. Do your linter lints js code?

Reply:
For some reason linter does not work correctly, will have to check it